neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -97,6 +100,8 @@ public virtual Fixed8 NetworkFee
 
         public virtual Fixed8 SystemFee => Settings.Default.SystemFee.TryGetValue(Type, out Fixed8 fee) ? fee : Fixed8.Zero;
 
+        public uint Difficult => (uint)BitConverter.ToInt32(Hash.ToArray().Take(4).Reverse().ToArray(), 0);
```

Comment:
This is nice Shargon! But it's assuming a standard difficulty of 4 zeroes... perhaps we just leave users compete for difficulty? More zeroes, more chance to get free tx.